client/app/components/packages/landuse-form/public-facilities.hbs

```diff
@@ -0,0 +1,340 @@
+              )
+            }}
+          />
+          <label for="dcpOfficespaceleaseopt">
+            Office Space Lease
+          </label>
+
+          <br>
+
+          <Input
+            type="checkbox"
+            id="dcpAcquisitionopt"
+            data-test-checkbox="dcpAcquisitionopt"
+            checked={{if
+              (eq
+                form.data.dcpAcquisitionopt
+                (optionset 'landuseForm' 'dcpAcquisitionopt' 'code' 'Yes')
+              ) "checked"}}
+            onClick={{fn (mut form.data.dcpAcquisitionopt)
+              (if form.data.dcpAcquisitionopt
+                (optionset 'landuseForm' 'dcpAcquisitionopt' 'code' 'No')
+                (optionset 'landuseForm' 'dcpAcquisitionopt' 'code' 'Yes')
+              )
+            }}
+          />
+          <label for="dcpAcquisitionopt">
+            Public Facility Acquisition
+          </label>
+
+          <br>
+
+          <Input
+            type="checkbox"
+            id="dcpSiteselectionopt"
+            data-test-checkbox="dcpSiteselectionopt"
+            checked={{if
+              (eq
+                form.data.dcpSiteselectionopt
+                (optionset 'landuseForm' 'dcpSiteselectionopt' 'code' 'Yes')
+              ) "checked"}}
+            onClick={{fn (mut form.data.dcpSiteselectionopt)
+              (if form.data.dcpSiteselectionopt
+                (optionset 'landuseForm' 'dcpSiteselectionopt' 'code' 'No')
+                (optionset 'landuseForm' 'dcpSiteselectionopt' 'code' 'Yes')
+              )
+            }}
+          />
+          <label for="dcpSiteselectionopt">
+            Public Facility Site Acquisition
+          </label>
+        </Ui::Question>
+
+        <Ui::Question
+        as |dcpIndicatetypeoffacilityQ|>
+          <dcpIndicatetypeoffacilityQ.Label>
+            What is the type of facility?
+          </dcpIndicatetypeoffacilityQ.Label>
+
+          <form.Field
+            @attribute="dcpIndicatetypeoffacility"
+            @type="radio-group"
+          as |DcpIndicatetypeoffacilityRadioGroup|>
+            <DcpIndicatetypeoffacilityRadioGroup
+              @options={{optionset 'landuseForm' 'dcpIndicatetypeoffacility' 'list'}}
+            />
+          </form.Field>
+        </Ui::Question>
+
+        <Ui::Question
+        as |dcpExistingfacilityproposedtoremainoptQ|>
+          <dcpExistingfacilityproposedtoremainoptQ.Label>
+            Will the existing facility remain?
+          </dcpExistingfacilityproposedtoremainoptQ.Label>
+
+          <form.Field
+            @attribute="dcpExistingfacilityproposedtoremainopt"
+            @type="radio-group"
+          as |DcpExistingfacilityproposedtoremainoptRadioGroup|>
+            <DcpExistingfacilityproposedtoremainoptRadioGroup
+              @options={{optionset 'landuseForm' 'dcpExistingfacilityproposedtoremainopt' 'list'}}
+              @onChange={{fn this.resetDependentFields this.dependantsOf.dcpExistingfacilityproposedtoremainopt}}
```

Comment:
interesting!

Reply:
Yep! This is to clear values of nested answers when this radio answer changes